src/containers/ByCharacterQuotes.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import QuoteList from '../components/app/quotes/QuoteList';
+import { findQuotesByCharacter } from '../services/futuramaApi';
+
+export default class ByCharacterQuotes extends Component {
+  static propTypes = {
+    match: PropTypes.shape({
+      params: PropTypes.shape({
+        character: PropTypes.string.isRequired,
+      }).isRequired,
+    }).isRequired
+  };
+
+  state = {
```

Comment:
this lab focused on hooks, so should refactor you state to something like useState()

Reply:
Keep this in mind for both containers that you have.